apps/web/lib/plain/plainChat.tsx

```diff
@@ -70,11 +70,23 @@ interface PlainChatConfig {
 
 const PlainChat = () => {
   const [config, setConfig] = useState<PlainChatConfig | null>(null);
+  const [isSmallScreen, setIsSmallScreen] = useState(false);
   const { data: session } = useSession();
   const pathname = usePathname();
   const searchParams = useSearchParams();
 
+  const isAppDomain = typeof window !== "undefined" && window.location.hostname === "app.cal.com";
```

Comment:
is it possible that window is undefined given "use client" is set? Not an issue just an observation

Reply:
how else can we check for the domain within the use effect? any recommendations?